source/GameAction.cpp

```diff
@@ -385,7 +385,8 @@ GameAction GameAction::Instantiate(map<string, string> &subs, int jumps, int pay
 		result.events[it.first] = make_pair(day, day);
 	}
 
-	result.giftShips = giftShips;
+	for(auto &&it : giftShips)
+		result.giftShips.emplace_back(it.Instantiate(subs));
```

Comment:
I'd change emplace_back to push_back, because Instantiate returns a complete ShipManager. Maybe also use std::move?
```suggestion
	for(auto &&it : giftShips)
		result.giftShips.push_back(std::move(it.Instantiate(subs)));
```

Reply:
I applied this suggestion manually and retested. It worked.

Reply:
Alas, it does not work. MacOS tests say this:

>   Error: /Users/runner/work/endless-sky/endless-sky/source/GameAction.cpp:464:30: error: moving a temporary object prevents copy elision [-Werror,-Wpessimizing-move]

Reply:
yeah, looks like some compilers don't like the move here, but you may keep the push_back
```suggestion
	for(auto &&it : giftShips)
		result.giftShips.push_back(it.Instantiate(subs));
```

Reply:
I applied that change manually and it works on Ubuntu.